### DISCUSSION TITLE
Any pitfalls upgrading from Fedora 38 to Fedora 39?

### DISCUSSION THREAD

**Discussion by bobossler** [JaKooLit/Fedora-Hyprland#47] [Q&A]
Just a curiousity question really. I repurposed an Asus Chromebook into a Linux laptop and installed a Fedora 38 spin. Works great! I Followed the Fedora-Hyprland install and the dotfiles install, and everything works even better. Awesome project here!!!

I have been considering an upgrade to Fedora 39, but am hesitant to break the fantastic setup I have now. There is no "need" or compelling reason to upgrade to 39 at the moment, just considering it.

My question is: are there any pitfalls or known issues performing the upgrade, that would break my existing Hyprland setup? My first thought is that there should not be, but so much care and effort went into the Fedora-Hyprland scripts, I am curious if the Fedora versio upgrade has any affect.

**Comment by JaKooLit** [comment]
I am not 100% sure but it should not have any issues. But dont quote me on it. When I created the script, I tested on 38 and 39 and both works.

Some even tried on rawhide. However, there is an issue building with new nwg-look from what I gather. I will check next month once I arrived home since atm, im at work and my data is very limited
